schema/aggregatable_report.schema.json

```diff
@@ -5,8 +5,8 @@
   "description": "An Attribution Reporting aggregatable report",
   "type": "object",
   "properties": {
-    "aggregation_coordinator_identifier": {
-      "type": "string"
+    "aggregation_coordinator_origin": {
```

Comment:
The "required" section below also needs to be updated.

Reply:
Updated, thanks.